nbconvert/preprocessors/execute.py

```diff
@@ -81,6 +81,40 @@ def from_cell_and_msg(cls, cell, msg):
 {ename}: {evalue}
 """
 
+class CellExecutionTimeoutError(TimeoutError):
+    """
+    Custom exception to propagate exceptions that are raised during
+    notebook execution to the caller for timeout. This is mostly
+    useful when using nbconvert as a library, since it allows to
+    deal with failures gracefully.
+    """
+    def __init__(self, traceback):
+        super(CellExecutionTimeoutError, self).__init__(traceback)
+        self.traceback = traceback
```

Comment:
I think you can remove this `__init__` - the traceback is not required for this exception